components/upload/UploadList.tsx

```diff
@@ -98,6 +98,27 @@ export default class UploadList extends React.Component<UploadListProps, any> {
     return icon;
   };
 
+  handleActionIconRender = (customIcon: React.ReactNode, callback: () => void, title?: string) => {
+    if(React.isValidElement(customIcon)) {
+      return React.cloneElement(customIcon,
+        {
+          onClick: (e: React.MouseEvent) => {
+            callback();
+            if (customIcon.props.onClick) {
+              customIcon.props.onClick(e);
+            }
+          },
+          title,
```

Comment:
use `customIcon.props` spread?

Reply:
Horrible mistake. Thank you for correction